packages/tables/resources/views/index.blade.php

```diff
@@ -326,24 +328,34 @@ class="shrink-0"
                         <x-slot name="header">
                             @if ($isReordering)
                                 <th></th>
-                            @elseif ($isSelectionEnabled)
-                                <x-tables::checkbox-cell>
-                                    <x-slot
-                                        name="checkbox"
-                                        x-on:click="toggleSelectRecordsOnPage"
-                                        x-bind:checked="
-                                            if (areRecordsSelected(getRecordsOnPage())) {
-                                                $el.checked = true
-
-                                                return 'checked'
-                                            }
-
-                                            $el.checked = false
-
-                                            return null
-                                        "
-                                    ></x-slot>
-                                </x-tables::checkbox-cell>
+                            @else
+                                @if (count($actions) && (! $isReordering) && $actionsPosition === Position::BeforeCells)
```

Comment:
We no longer need the `! $isReordering` check, right? Remove the other occurrences as well where suitable.